content/kubevirt/index.md

```diff
@@ -158,13 +160,64 @@ podman push ${REGISTRY}/cnv/iis:latest
 oc apply -f https://raw.githubusercontent.com/openshift-examples/web/master/content/kubevirt/iis-template.yaml
 ```
 
+## Resource Capacity Calculation
+
+It’s the same as any other OpenShift sizing. Each VM has a request for the full amount of memory plus overhead. **There is no memory overcommitment (yet)**. Memory will be your biggest constraint and is how the cluster/nodes will be sized for the majority of deployments.
+
+At a high level, the process is to determine the amount of virtualization resources needed (VM sizes, overhead, burst capacity, failover capacity), add that to the amount of resources needed for cluster services (logging, metrics, ODF/ACM/ACS if hosted in the same cluster, etc.) and customer workload (hosted control planes, other Pods deployed to the hardware, etc.), then find a balance of node size vs node count.
+
+### CPU capacity calculation
```

Comment:
Stupid question, what's the value to copy the calculations from
official docs?

Reply:
I see it valuable when you want to have it quickly by hand if needed (like in conversations). Do you see the whole section not relevant?

Reply:
Got it. Make sense. Maybe you can add a link to the official docs.redhat.com.